controls/bsi_app_4_4.yml

```diff
@@ -217,9 +217,12 @@ controls:
         • Logging of changes
         • Regular data backups.
     notes: >-
-      TBD
-    status: pending
-    rules: []
+      This requirement needs to be adressed in the respective separate systems.
+      However, one requirement can be checked automated: Encrypted communication all image registries.

```

Comment:
Suggestion for line 221, since the grammar is a little bit off (bold from me to say that :D) :
`However, one requirement (Encrypted communication on all network ports) can partitially be checked by ensuring that no registry is allowed in over insecure protocols`

Reply:
Thanks for rewording, I took your version.